tensorflow-sources.file

```diff
@@ -118,9 +120,15 @@ cp -p $srcdir/compiler/tf2xla/lib*.so* $libdir/
 cp -p $srcdir/compiler/xla/lib*.so* $libdir/
 
 for l in tensorflow_cc tensorflow_framework tensorflow ; do
+  # link from realversion to majorversion
+  if [ ! -e $libdir/lib${l}.so.%{majorversion} ] ; then
+    [ -e $libdir/lib${l}.so.%{realversion} ] || exit 1
+    (cd $libdir && ln -s lib${l}.so.%{realversion} lib${l}.so.%{majorversion})
+  fi
+  # link from majorversion to default lib
   if [ ! -e $libdir/lib${l}.so ] ; then
-    [ -e  $libdir/lib${l}.so.%{realversion} ] || exit 1
-    ln -s lib${l}.so.%{realversion} $libdir/lib${l}.so
```

Comment:
while you are at it, may be delete `$libdir/lib${l}.so.%{majorversion}` and always create symlink to `lib${l}.so.%{realversion}`. This will avoid have duplicate files  [a]

By the way, there is no need to do `cd  $libdir ; ln ...` , `ln -s lib${l}.so.%{realversion} $libdir/lib${l}.so` should work :-)

[a]
```
-r-xr-xr-x 1 cmsbld cmsbld      7664 Dec 16 15:15 libcpu_function_runtime.so
-r-xr-xr-x 1 cmsbld cmsbld       112 Dec 16 15:15 libcpu_function_runtime.so-2.params
lrwxrwxrwx 1 cmsbld cmsbld        25 Dec 16 15:26 libtensorflow_cc.so -> libtensorflow_cc.so.2.0.0
-r-xr-xr-x 1 cmsbld cmsbld 161379080 Dec 16 15:02 libtensorflow_cc.so.2.0.0
-r-xr-xr-x 1 cmsbld cmsbld    312319 Dec 16 15:02 libtensorflow_cc.so.2.0.0-2.params
-r-xr-xr-x 1 cmsbld cmsbld  36165184 Dec 16 14:46 libtensorflow_framework.so
-r-xr-xr-x 1 cmsbld cmsbld  36165184 Dec 16 14:46 libtensorflow_framework.so.2
-r-xr-xr-x 1 cmsbld cmsbld  36165184 Dec 16 14:46 libtensorflow_framework.so.2.0.0
-r-xr-xr-x 1 cmsbld cmsbld    169854 Dec 16 14:35 libtensorflow_framework.so.2.0.0-2.params
lrwxrwxrwx 1 cmsbld cmsbld        22 Dec 16 15:26 libtensorflow.so -> libtensorflow.so.2.0.0
-r-xr-xr-x 1 cmsbld cmsbld 139098128 Dec 16 15:01 libtensorflow.so.2.0.0
-r-xr-xr-x 1 cmsbld cmsbld    311082 Dec 16 15:01 libtensorflow.so.2.0.0-2.params
-r-xr-xr-x 1 cmsbld cmsbld    107112 Dec 16 15:13 libtf2xla.so
-r-xr-xr-x 1 cmsbld cmsbld        87 Dec 16 15:12 libtf2xla.so-2.params
-r-xr-xr-x 1 cmsbld cmsbld     13696 Dec 16 15:15 libxla_compiled_cpu_function.so
-r-xr-xr-x 1 cmsbld cmsbld       125 Dec 16 15:15 libxla_compiled_cpu_function.so-2.params
```

Reply:
Sounds good